pkg/webhook/workspace/config.go

```diff
@@ -31,6 +38,11 @@ func Configure(ctx context.Context) error {
 		return err
 	}
 
+	saUID, err := controllerUID(ctx, c)
```

Comment:
Consider moving in below `if !server.IsSetUp()` condition to make that env var optional when it's not used - local operator run, or when webhooks are disabled.

Reply:
Done.